storage/aptosdb/src/pruner/mod.rs

```diff
@@ -39,10 +39,10 @@ use worker::{Command, Worker};
 pub(crate) struct Pruner {
     /// DB version window, which dictates how many versions of state store
     /// to keep.
-    state_store_prune_window: Version,
+    state_store_prune_window: Option<Version>,
```

Comment:
Not directly related to your change, but I guess the window should be represented as something like usize or u64 rather than Version? (although they might actually be the same type)

Reply:
Seems like they are the same https://github.com/aptos-labs/aptos-core/blob/15c86baf0cd11e1d485832fc8b488ed77348b034/types/src/transaction/mod.rs#L56

Reply:
I meant conceptually they should be different. IIUC window is the difference between two versions.
(so version v.s. window is similar to timestamp v.s. duration)

Reply:
Can I address this in a separate PR?

Reply:
Sure.